apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -692,36 +608,6 @@ export class DetailViewContents extends React.Component {
     );
   };
 
-  renderRegistrationLinks = () => {
```

Comment:
This entire method wasn't being used –– see https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx for original file

Reply:
Nice catch!